themis.podspec

```diff
@@ -1,19 +1,24 @@
 Pod::Spec.new do |s|
     s.name = "themis"
-    s.version = "0.13.1"
+    s.version = "0.13.2"
     s.summary = "Data security library for network communication and data storage for iOS and mac OS"
     s.description = "Themis is a convenient cryptographic library for data protection. It provides secure messaging with forward secrecy and secure data storage. Themis is aimed at modern development practices and has a unified API across 12 platforms, including iOS/macOS, Ruby, JavaScript, Python, and Java/Android."
     s.homepage = "https://cossacklabs.com"
     s.license = { :type => 'Apache 2.0'}
 
     s.source = { :git => "https://github.com/cossacklabs/themis.git", :tag => "#{s.version}" }
+  
     s.author = {'cossacklabs' => 'info@cossacklabs.com'}
 
     s.module_name = 'themis'
-    s.default_subspec = 'openssl-1.1.1'
-
-    s.ios.deployment_target = '8.0'
-    s.osx.deployment_target = '10.9'
+    
+    # TODO(vixentael, 11 oct 2020): as xcode12 introduces new arm64 architecture, our own openssl framework doesn't work yet
+    # Change openssl-1.1.1 to default when fix our openssl
+    #s.default_subspec = 'openssl-1.1.1'
+    s.default_subspec = 'themis-openssl'
```

Comment:
How about dropping the entire `openssl-1.1.1` subspec from this hotfix, considering that it does not work?